docs/service-catalog/06-01-service-binding-usage.md

```diff
@@ -3,15 +3,17 @@ title: ServiceBindingUsage
 type: Custom Resource
 ---
 
-The `servicebindingusages.servicecatalog.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to inject Secrets to the application. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+The `servicebindingusages.servicecatalog.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to inject Secrets to an application. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
 
 ```
 kubectl get crd servicebindingusages.servicecatalog.kyma-project.io -o yaml
 ```
 
 ## Sample custom resource
 
-This is a sample resource in which the ServiceBindingUsage injects a Secret associated with the `redis-instance-binding` ServiceBinding to the `redis-client` Deployment in the `production` Namespace. This example has the **conditions.status** field set to `true`, which means that the ServiceBinding injection is successful. If this field is set to `false`, the **message** and **reason** fields appear.
+This is a sample resource in which the ServiceBindingUsage injects a Secret associated with the `redis-instance-binding` ServiceBinding to the `redis-client` Deployment in the `production` Namespace. In this example, the **status.conditions.status** field is set to `True`, which means that the ServiceBinding injection is successful. If the injection fails, this field is set to `False` and the **message** and **reason** fields appear. This example has also the **envPrefix.name** field specified, which adds a prefix to all environment variables injected from a given Secret to your Pod. This allows you to separate environment variables injected from different Secrets. By default, the prefixing is disabled. Set the **envPrefix.name** to enable it.
+
+>**NOTE:** The prefix itself is not separated from the name of an environment variable by any character. If you want to separate your prefix, add a special character at the end of it.
```

Comment:
add an example rather than this long sentence which none wants to read........

Reply:
Example added